index.html

```diff
@@ -158,7 +267,7 @@ <h5>About Myself</h5>
                         <img class="lang_head_icon" src="downarrow.svg">
                         <h6>Languages</h6>
                     </div>
-                    <ul class="languange_list">
+                    <ul class="language_list">
```

Comment:
- [ ] As stated by the previous reviewer You did a good job implementing the desktop version good job so far 👍 but there is something you are missing out. kindly consider going back to your mobile version and fixing it all cause it was messed up when developing the desktop version remember we have to have a desktop version after the mobile version is okay and it has to stay okay to meet the responsiveness requirements while doing so consider also removing the horizontal scroll in your mobile version design doing all the above will help you to meet the below requirement

![image](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/83360636/6422fbf1-1cfb-428b-a066-1d0e5a150b83)